docs/components/navs.md

```diff
@@ -14,7 +14,11 @@ Navigation available in Bootstrap share general markup and styles, from the base
 
 ## Base nav
 
-The base `.nav` component is built with flexbox and provide a strong foundation for building all types of navigation components. It includes some style overrides (for working with lists), some link padding for larger hit areas, and basic disabled styling. No active states are included in the base nav.
+The base `.nav` component is built with flexbox and provide a strong foundation for building all types of navigation components. It includes some style overrides (for working with lists), some link padding for larger hit areas, and basic disabled styling.
+
+{% callout info %}
+The base `.nav` component does not include any `.active` state. The following examples include the class, mainly to demonstrate that this particular class does not trigger any special styling.
```

Comment:
@mdo are you ok with me just leaving this one callout in (and neutering the others)?